### PR TITLE
Replaced processing state of services notification with purpose built processing notification

### PR DESCRIPTION
**Pull request by EmmanuelMess**
Replaced processing state of services notification with purpose built processing notification. So that the notification doesn't show "unknown" fields.

@VishalNehra do check it out and tell me what you think

Don't merge yet, haven't really tested.

Known bugs:
* Flickering not fixed, the flickering seems to be due to a creation of the same Service quite a few times.

### CODE REVIEW

**Comment by EmmanuelMess** [comment]
You can now test.

**Comment by VishalNehra** [comment]
I really don't think this is a good idea. Too much inconsistent notification when it gets halted in between from time to time (on slow connections), the bigView changes to smallView of the processing notification.
Plus, the layout itself isn't much pleasing...
screenshot_amaze_20180708-025003

**Comment by EmmanuelMess** [comment]
I need more constructive ideas...

File: app/src/main/java/com/amaze/filemanager/asynchronous/services/CopyService.java [hunk 1]
```diff
@@ -198,6 +200,26 @@ protected void setPercentProgress(float progress) {
...
     }
 
+    @Override
+    protected ProcessingNotificationBuilder getProcessingNotificationBuilder() {
+        String filenameText;
+
+        if(fileAmount == 1) {
+            filenameText = firstFilename;
+        } else if(fileAmount == 2) {
+            filenameText = c.getString(R.string.processing_notif_two_filenames, firstFilename, secondFilename);
```
Comment (EmmanuelMess) [inline app/src/main/java/com/amaze/filemanager/asynchronous/services/CopyService.java]: Self note: see of using `TextUtils.listEllipsize`.

**Pull request closed without merging**